src/components/ClubHome/Daily/Table.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styles from "../../../styles/Club/Home/Schedule/Calendar.module.scss";
+
+const Table = ({ calendarArr }) => {
+  return (
+    <table className={styles.calendar}>
+      <tbody>
+        <tr className={styles.days}>
+          <td className={styles.daysblock}>
+            <span>일</span>
+          </td>
+          <td className={styles.daysblock}>
+            <span>월</span>
+          </td>
+          <td className={styles.daysblock}>
+            <span>화</span>
+          </td>
+          <td className={styles.daysblock}>
+            <span>수</span>
+          </td>
+          <td className={styles.daysblock}>
+            <span>목</span>
+          </td>
+          <td className={styles.daysblock}>
+            <span>금</span>
+          </td>
+          <td className={styles.daysblock}>
+            <span>토 </span>
+          </td>
```

Comment:
```suggestion
{weekDays.map((day, index) => {
   <td key={index} className={styles.daysblock}>
            <span>{day}</span>
   </td>
})
```
이렇게 해야되는건 아니지만 이렇게도 할 수 있을거 같아요